src/olympia/accounts/tests/test_tasks.py

```diff
@@ -80,6 +81,12 @@ def test_success_basic(self, delete_picture_mock):
         assert not Collection.objects.filter(id=collection.id).exists()
         assert not another_addon.ratings.all().exists()
         delete_picture_mock.assert_called()
+        # Two UserLog instances get created - one for request.user, one for the instance
```

Comment:
Uh ? Why ? I'd understand if the users were different (and even then, it could be confusing to admins looking at the data later, so we should make sure we can easily tell the 2 apart), but that's not the case here ?

Reply:
One `UserLog` instance is always created for the user adding the activity log; because this is related to a `UserProfile` object another `UserLog` is created for that (it'd be an `AddonLog` if it related to an `Addon`, etc).  

It's existing activity log functionality but I could change it?

Reply:
https://github.com/mozilla/addons-server/issues/18811 to drop `UserLog` entirely